numba/np/arrayobj.py

```diff
@@ -1562,6 +1563,36 @@ def array_T(context, builder, typ, value):
     return impl_ret_borrowed(context, builder, typ, res)
 
 
+@overload(np.rot90)
+def numpy_rot90(arr, k=1):
+    # supporting axes argument it needs to be included in np.flip
+    if not isinstance(k, (int, types.Integer)):
+        raise errors.TypingError('The second argument "k" must be an integer')
+    if not isinstance(arr, types.Array):
+        raise errors.TypingError('The first argument "arr" must be an array')
+
+    if arr.ndim < 2:
+        raise ValueError('Input must be >= 1-d.')
+
+    axes_list = (1, 0, *range(2, arr.ndim))
+
+    def impl(arr, k=1):
+        arr = np.asarray(arr)
```

Comment:
Is this needed if I check for `types.Array` in line 1571? Tests passes without this line.

Reply:
I think that can indeed be removed.

Reply:
Ok, I removed it.